### DISCUSSION TITLE
Mouse move/resize compatibility

### DISCUSSION THREAD

**Discussion by freisenhauer** [nikitabobko/AeroSpace#656] [feature-ideas]
I don't know, if this was mentioned before, i couldn't find it anywhere. I used to use yabai as tiling window manager and there was a feature to move or resize windows on a workspace by holding a modifier and holding left (move) or right (resize) click. If i use the default macos builtin solution (at the corners or the title bar), this works with aerospace. But if I use a tool like Easy Move + Resize (https://github.com/dmarcotte/easy-move-resize), this is just flickering around and doing nothing at the end of the day. Is there any reason, why this is not working ? Is it a bug or on purpose ? Would it be possible to either make it compatible to external move and resize toolings or just build it into aerospace ? 

I'm using a single 4k monitor and I always have multiple windows on screen and am resizing them in both directions (horizontal and vertical) with the mouse was always faster than shortcuts to resize dimensions independently.

**Comment by titilambert** [comment]
https://github.com/dmarcotte/easy-move-resize/issues/111

**Comment by Shahor** [comment]
Hi 👋🏽 

Is this something the team would be considering? Coming from Yabai and heavily using those to roughly resize and move stuff around, it makes the transition a bit rough.

I was thinking about actually having yabai on top of aerospace just for that feature only but I'm not sure how those would cohabit 😅

**Comment by zhenpewu** [comment]
it this what guys looking for ?


https://nikitabobko.github.io/AeroSpace/goodies#move-by-dragging-any-part-of-the-window
```
defaults write -g NSWindowShouldDragOnGesture -bool true
```

Now, you can move windows by holding ctrl + cmd and dragging any part of the window (not necessarily the window title)

**Reply by Shahor** [reply]
> it this what guys looking for ?
> 
> https://nikitabobko.github.io/AeroSpace/goodies#move-by-dragging-any-part-of-the-window
> 
> ```
> defaults write -g NSWindowShouldDragOnGesture -bool true
> ```
> 
> Now, you can move windows by holding ctrl + cmd and dragging any part of the window (not necessarily the window title)

Ah, thank you for that! 
Although it scratches part of the itch it's not exactly what I was looking for :(

1. It doesn't allow me to use my own bindings
2. It only allows moving but I also want resizing
3. It's not working 😬 but that might be because I need to reboot and I haven't yet.
